typescript/sdk/src/providers/MultiProvider.ts

```diff
@@ -21,38 +21,58 @@ import { pick } from '../utils/objects';
 type Provider = providers.Provider;
 
 export class MultiProvider {
+  private readonly logger: Debugger = debug('hyperlane:MultiProvider');
```

Comment:
Ohh great idea, wonder (optional nit) we should have an options param in the constructor that allows users to override this